test/starkinfra/test_pixinfraction.rb

```diff
@@ -71,24 +71,25 @@
   end
 
   it 'page and update' do
-    infraction = StarkInfra::PixInfraction.get(get_infraction('reported'))
+    infraction = StarkInfra::PixInfraction.get(get_infraction('fraud'))
```

Comment:
Aqui tem que fazer o filtro pro flow e por status. Você só pode fazer o update (responder) infractions que têm status de delivered e flow in. Type não tem nd a ver.

Reply:
mesma coisa para PixChargeback